frontend/mobilityai_react/App/Screens/GetDate.js

```diff
@@ -1,44 +1,71 @@
 import React, { Component } from 'react';
-import { View, Button, StyleSheet } from 'react-native';
-import DatePicker from 'react-native-datepicker'
+import { Text, View, Button, StyleSheet, TouchableOpacity } from 'react-native';
 
+import DateTimePicker from 'react-native-modal-datetime-picker';
 
- // Getting the current date
- export default class GetDate extends Component {
-    constructor(props) {
-        super(props);
 
-        const date = new Date().toLocaleDateString('en-CA');
+// Getting the current date
+export default class GetDate extends Component {
+    state = {
+        isDateTimePickerVisible: false,
+    };
 
+    _showDateTimePicker = () => this.setState({ isDateTimePickerVisible: true });
+    _hideDateTimePicker = () => this.setState({ isDateTimePickerVisible: false });
+    _handleDatePicked = (date) => {
+        console.log('A date has been picked: ', date);
+        this._hideDateTimePicker();
+        this.props.date(date);
+        this.setState({date: date});
+        var fDate = this.getFormattedDate(this.state.date);
+        this.setState({formattedDate: fDate});
+    };
+
+    getFormattedDate(date) {
```

Comment:
Can we use [moment.js](https://momentjs.com/) for the date formatting, no need to write it from scratch.